oci/dir.go

```diff
@@ -0,0 +1,142 @@
+package oci
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/opencontainers/go-digest"
+)
+
+// Directory is ImageDriver for directory representation of an OCI image.
+// The directory must not exist before calling Init function of this structure.
+func Directory(path string) ImageDriver {
```

Comment:
I thought we were removing this.

It would be good to put together a proposal for directory based layouts. This use case really doesn't make sense. The OCI layout is meant for an archive.